core/src/main/java/io/questdb/PropServerConfiguration.java

```diff
@@ -1099,6 +1100,7 @@ public PropServerConfiguration(
             this.sqlWindowTreeKeyPageSize = Numbers.ceilPow2(getIntSize(properties, env, PropertyKey.CAIRO_SQL_WINDOW_TREE_PAGE_SIZE, sqlWindowTreeKeyPageSize));
             int sqlWindowTreeKeyMaxPages = getInt(properties, env, PropertyKey.CAIRO_SQL_ANALYTIC_TREE_MAX_PAGES, Integer.MAX_VALUE);
             this.sqlWindowTreeKeyMaxPages = getInt(properties, env, PropertyKey.CAIRO_SQL_WINDOW_TREE_MAX_PAGES, sqlWindowTreeKeyMaxPages);
+            this.tempSqlOperatorPrecedence = getBoolean(properties, env, PropertyKey.TEMP_CAIRO_SQL_OPERATOR_PRECEDENCE, false);
```

Comment:
Hm...yes, `true` = new precedence, you are right. But I don't want to introduce breaking change in this PR - so want to make old behaviour default one...

Reply:
I agree with Nick: the default should be breaking behavior. Also, it may be a good idea to name the prop something like `cairo.sql.legacy.operator.precedence`.

Reply:
Ok, removed todos from here. Leaved log as is - not sure that queryId will be actually helpful...